checkov/common/bridgecrew/platform_integration.py

```diff
@@ -452,6 +454,15 @@ def persist_run_metadata(self, run_metadata: dict[str, str | list[str]]) -> None
             return
         persist_run_metadata(run_metadata, self.s3_client, self.bucket, self.repo_path)
 
+    def persist_logs_stream(self, logs_stream: StringIO) -> None:
```

Comment:
@Saarett I add new functions to handle the support flag.

Reply:
I'm a little afraid of implementing here function that named "persist_logs_stream" while you also import it from other file.
1. did you checked that it behaves as expected?
2. unless you use the same names on purpose (maybe there is some pattern that I'm not familiar with), I suggest to use different names

Reply:
nevermind. my bad. Now I noticed that the current method is class-one. it is ok